lib/util/endpoints.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+module.exports.getSignalingEndpoint = (environment, region) => {
```

Comment:
@makarandp0 We can refactor the VSG hostname calculation logic into its own function:
```js
function getVsgHostName(environment, region) {
  region = region === 'gll' ? 'global' : encodeURIComponent(region);
  return environment === 'prod'
    ? `wss://${region}.vss.twilio.com`
    : `wss://${region}.vss.${environment}.twilio.com`
}
```
Then, `getSignalingEndpoint` and `getInsightsEndpoint` will become:
```js
function getInsightsEndpoint(environment, region) {
  return `${getVsgHostName(environment, region)}/insights`;
}

function getSignalingEndpoint(environment, region) {
  return `${getVsgHostName(environment, region)}/signaling`;
}

exports.getInsightsEndpoint = getInsightsEndpoint;
exports.getSignalingEndpoint = getSignalingEndpoint;
```